CMakeLists.txt

```diff
@@ -333,7 +333,7 @@ endif(WITH_VALGRIND_INSTRUMENTATION)
 # MPI
 #
 
-find_package(MPI 3.0 REQUIRED)
+find_package(MPI 2.0 REQUIRED)
```

Comment:
we do actually use some MPI 3 functions. Perhaps Microsoft MPI has partial MPI 3 support, otherwise you would be seeing compile errors?